extensions/eo/README.md

```diff
@@ -14,8 +14,9 @@ natural focus, and encourage other sensors to make their own extensions. Once th
 these fields will evolve to higher level extensions. In the meantime other implementations are welcome
 to reuse the names and definitions here.
 
-It is not necessary, but recommended to use the [Commons extension](../commons/README.md)
-(see chapter "Placing common fields in Collections").
+A lot of EO data will have common metadata across many Items. It is not necessary, but recommended	
+to place common fields in [STAC Collections](../../collection-spec/collection-spec.md).
```

Comment:
Perhaps link to collection-spec.md#common-fields-and-standalone-collections so it takes people directly to the relevant section?